src/nvim/ops.c

```diff
@@ -3535,11 +3534,8 @@ int do_join(long count,
   del_lines(count - 1, FALSE);
   curwin->w_cursor.lnum = t;
 
-  /*
-   * Set the cursor column:
-   * Vi compatible: use the column of the first join
```

Comment:
have we removed (or decided to) this compatibility option? otherwise this comment should be kept